Better Domenstration

Longyu975 opened the pull request:
Update parameters and video to README

Longyu975 committed:
Update parameters to README

Longyu975 committed:
Fixed font

Longyu975 committed:
Fixed table

Longyu975 committed:
Add video demonstrations to README

Added video demonstrations for periodic and reflective boundary conditions.

Longyu975 committed:
Fix video links and section title in README

Updated video links to remove markdown syntax and corrected section title.

Longyu975 committed:
Swap video sources in README.md

rogeriojorge commented:
Looks good! One thing for the videos:
- I took away the LogNorm for the phase space; I believe it looks better without it. Can you try it? It is in the rj/plotting branch
- Can you remove the charge conservation line? It makes too many orders of magnitude appear in the plot
- Can you move the movies up so that people can see it as one of the first few things after logo/badges?

Longyu975 committed:
Fix input file path and update video links in README

Updated example input file path and replaced video links for boundary conditions.

rogeriojorge reviewed (commented):
It seems that the movies of periodic/reflective are swapped.

Also, for the input parameters, can you mention that these are the simple ones along x for an electron/ion plasma?

rogeriojorge commented:
You can then mention that a more detailed list of parameters can be found in the bump on tail example.

Longyu975 committed:
Enhance README with bump-on-tail demonstration

Added a demonstration section for the bump-on-tail example, including periodic and reflective boundary condition videos.

Longyu975 committed:
Strike out completed Task 6 in README
